app/routers/city.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from .. import crud, schemas, database
+
+router = APIRouter()
+
+@router.post("/cities/", response_model=schemas.City)
+def create_city(
+        city: schemas.CityCreate,
+        db: Session = Depends(database.get_db)
+):
+    return crud.create_city(db=db, city=city)
+
+@router.get("/cities/", response_model=List[schemas.City])
+def read_cities(
+        skip: int = 0,
+        limit: int = 10,
+        db: Session = Depends(database.get_db)
+):
+    return crud.get_cities(db, skip=skip, limit=limit)
+
+@router.delete("/cities/{city_id}", status_code=204)
```

Comment:
The `status_code=204` in the `@router.delete` decorator is correct for a successful deletion, but ensure that the function returns a response with a 204 status code explicitly, as returning `None` might not automatically set this status code in some configurations.